.vscode/launch.json

```diff
@@ -0,0 +1,23 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+
+        {
+            "name": "Python: Current File",
+            "type": "python",
+            "request": "launch",
+            "program": "${file}",
+            "console": "integratedTerminal",
+            "args": [
+              //  "--mqtt", "192.168.0.101", 
+                "--user", "duchesne.cedric@gmail.com", 
+                "--attributes", "/home/pi/dev/bmw_cd_exporter/attributes.json",
+                "--password","Nou2Velle/Experience",
```

Comment:
PLEASE DELETE THIS LINE AND CHANGE YOUR PASSWORD STRAIGHTAWAY

Reply:
See Github's FAQ on how to get it removed properly - https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/removing-sensitive-data-from-a-repository